docs/development/test.html

```diff
@@ -65,5 +48,9 @@ <h2>Parser Generator for JavaScript</h2>
         </footer>
     </div>
 </body>
+<script src='/js/test-bundle.min.js'></script>
+<script class="mocha-exec">
+  mocha.run();
+</script>
```

Comment:
I strongly advise that you consider `jest` rather than `mocha` as a test runner

`mocha` is largely dead and its coverage is quite seriously broken

I'm willing to do the conversion

I just converted a set of >2,000 `ava` tests to `jest` [a few months ago](https://github.com/StoneCypher/jssm/pull/417), without using codemods.  I understand how much work I'm getting myself into by offering this.

It's not worth blocking this PR, so, [I made another post-PR issue](https://github.com/peggyjs/peggy/issues/76).